src/export-api-v2.ts

```diff
@@ -575,48 +1003,103 @@ async function getManaPriceUsdWithRetry(retries = 3, delayMs = 1000): Promise<nu
     },
     {
       name: 'ESTATE DCL Marketplace Sales Fee',
-      description: 'Funds corresponding to the 2.5% fee applied to every ESTATE transaction (Minting or secondary)',
+      description:
+        'Funds corresponding to the 2.5% fee applied to every ESTATE transaction (Minting or secondary)',
       value: landEstate.estate.usdTotal || 0
     },
     {
       name: 'LAND  DCL Marketplace Sales Fee',
-      description: 'Funds corresponding to the 2.5% fee applied to every LAND transaction (Minting or secondary)',
+      description:
+        'Funds corresponding to the 2.5% fee applied to every LAND transaction (Minting or secondary)',
       value: landEstate.land.usdTotal || 0
     },
     {
       name: 'Wearables Minting Fee',
-      description: 'Funds corresponding to the 2.5% fee applied to Wearables minting on Polygon network via the Decentraland Marketplace',
+      description:
+        'Funds corresponding to the 2.5% fee applied to Wearables minting on Polygon network via the Decentraland Marketplace',
       value: polygonRow.usdTotal || 0
     },
     {
       name: 'DAO Wallets Transactions',
-      description: 'Transactions between the DAO Treasury and the DAO Committee wallets (e.g. Transaction gas refunds)',
-      value: prevMap['DAO Wallets Transactions'] || 0
+      description:
+        'Transactions between the DAO Treasury and the DAO Committee wallets (e.g. Transaction gas refunds)',
+      value: prevIncomeMap['DAO Wallets Transactions'] || 0
     },
     {
       name: 'LAND Rentals Marketplace Fee',
-      description: 'Funds corresponding to the 2.5% fee applied to every LAND or ESTATE rental transaction',
+      description:
+        'Funds corresponding to the 2.5% fee applied to every LAND or ESTATE rental transaction',
       value: rentalsRow.usdTotal || 0
     },
     {
       name: 'NAME DCL Marketplace Sales Fee',
-      description: 'Funds corresponding to the 2.5% fee applied to every NAME transaction (Minting or secondary)',
-      value: prevMap['NAME DCL Marketplace Sales Fee'] || 0
+      description:
+        'Funds corresponding to the 2.5% fee applied to every NAME transaction (Minting or secondary)',
+      value: prevIncomeMap['NAME DCL Marketplace Sales Fee'] || 0
     }
   ]
 
+    const previousIncome = Number(base?.income?.previous) || 0
+    const totalIncome =
+      typeof base?.income?.total === 'number'
```

Comment:
is this if ok ? if its defined we are using always the previous number defined 🤔